src/adapters/EasyRtcAdapter.js

```diff
@@ -54,22 +57,46 @@ class EasyRtcAdapter {
     this.easyrtc.setPeerListener(messageListener);
   }
 
+  updateTimeOffset() {
+    const clientSentTime = Date.now();
+
+    return fetch(document.location.href)
```

Comment:
Can we fetch a HEAD request instead of a GET request?

Reply:
Also not sure how this would react if the page happens to be cached. We may want to force a `no-cache`.